packages/product-feed-heureka/src/Model/Product/HeurekaProductDomain.php

```diff
@@ -58,7 +59,7 @@ public function __construct(HeurekaProductDomainData $heurekaProductDomainData)
     public function edit(HeurekaProductDomainData $heurekaProductDomainData)
     {
         $this->product = $heurekaProductDomainData->product;
-        $this->cpc = $heurekaProductDomainData->cpc;
+        $this->cpc = $heurekaProductDomainData->cpc !== null ? Money::fromValue($heurekaProductDomainData->cpc) : null;
```

Comment:
I would appreciate to have `$this->setCpc()` protected method as entities above have. But it is not a blocking point, just to avoid duplication ;-)

Reply:
The same applies for the `ZboziProductDomain` below

Reply:
This DRY violation will be removed when `Money::fromValue` gets removed. No worries :slightly_smiling_face: